go/service/gregor.go

```diff
@@ -225,7 +202,7 @@ func newGregorHandler(g *globals.Context) *gregorHandler {
 func (g *gregorHandler) Init() {
 	// Attempt to create a gregor client initially, if we are not logged in
 	// or don't have user/device info in G, then this won't work
-	if err := g.resetGregorClient(); err != nil {
+	if err := g.resetGregorClient(context.TODO()); err != nil {
```

Comment:
`Background()`?

Reply:
This isn't really the background, we just don't have a context. I thought `TODO` conveyed that it is just incomplete.